features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -896,8 +906,6 @@ internal open class RumViewScope(
         pendingActionCount++
         val rumContext = getRumContext()
 
-        val globalAttributes = GlobalRumMonitor.get(sdkCore).getAttributes().toMutableMap()
```

Comment:
should we still keep a local copy? like `val globalAttributes = globalAttributes.toMap()`.

Event will be created on another thread, and by the time it will be created `globalAttributes` may have some other value already, so it is better to have a local copy.